pmm_qa/Makefile

```diff
@@ -0,0 +1,11 @@
+default: help
+
+help:                       ## Display this help message.
+	@echo "Please use \`make <target>\` where <target> is one of:"
+	@grep '^[a-zA-Z]' $(MAKEFILE_LIST) | \
+	awk -F ':.*?## ' 'NF==2 {printf "  %-26s%s\n", $$1, $$2}'
+
+init:                 ## Create venv and install dependencies
```

Comment:
Please also add a init-ubuntu target to MakfFile:

init-ubuntu:
	sudo apt update
	sudo apt install -y python3.10-venv